### PR TITLE
Feature/sql standalone basic initial

### PR DESCRIPTION
**Pull request by dieppa**

### CODE REVIEW

File: mongoDB/springboot-mysql-annotation-basic/src/main/java/com/github/cloudyrock/mongock/examples/changelogs/ClientUpdaterChangeLog.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class ClientUpdaterChangeLog {
+
+    @ChangeSet(id = "data-updater-1", order = "001", author = "mongock")
+    public void dataInitializer1(@NonLockGuarded JdbcTemplate template) {
+        template.update("INSERT INTO client(name, email) values('name2', 'email2')");
+    }
+
+    @ChangeSet(id = "data-updater-2", order = "002", author = "mongock")
+    public void dataInitializer2(@NonLockGuarded JdbcTemplate template) {
+//        if (true) throw new RuntimeException("Expected exception");
```
Comment (osantana85) [inline mongoDB/springboot-mysql-annotation-basic/src/main/java/com/github/cloudyrock/mongock/examples/changelogs/ClientUpdaterChangeLog.java:18]: Why is commented?

File: mongoDB/springboot-mysql-annotation-basic/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/mysql
+spring.datasource.username=root
+spring.datasource.password=password
+spring.datasource.driver-class-name =com.mysql.jdbc.Driver
+#spring.jpa.show-sql: true
```
Comment (osantana85) [inline mongoDB/springboot-mysql-annotation-basic/src/main/resources/application.properties:6]: Why is commented?

**Review by osantana85 — APPROVED** [review]

File: mongoDB/springboot-mysql-annotation-basic/src/main/java/com/github/cloudyrock/mongock/examples/changelogs/ClientUpdaterChangeLog.java (hunk 1 above)
Comment (dieppa) [reply mongoDB/springboot-mysql-annotation-basic/src/main/java/com/github/cloudyrock/mongock/examples/changelogs/ClientUpdaterChangeLog.java:18]: I was testing the rollback per changelog. I could remove the entire changeset, Iguess

File: mongoDB/springboot-mysql-annotation-basic/src/main/resources/application.properties (hunk 2 above)
Comment (dieppa) [reply mongoDB/springboot-mysql-annotation-basic/src/main/resources/application.properties:6]: I will remove it

**Pull request merged**